posts/2023/unstructured-grid-collab-1.md

```diff
@@ -0,0 +1,106 @@
+A lucky overlap with the Improving Scientific Software conference, meant that collaborators from the DOE
+were also able to attend in-person.
+
+```{image} ../images/unstructured-grid-collab-1.jpg
+:alt: Photo of unstructured grids collaborative work time session.
+:width: 70%
+:align: center
+```
+
+## Plan
+
+Participants were asked to fill out a brief survey, and asked to self-organize by adding a one slide description of a project
+to a [presentation](https://docs.google.com/presentation/d/1UDjNMMCyXcHRT6BvX_Lcs4AEoHpNM8LntEfxWOWwwNQ/edit#slide=id.p) in the weeks preceding the event.
+
+At the event, we broke out in to approximately four groups. A recap from each group follows.
+
+## Recap
+
+#### Orhan Eroglu (CISL):
+
+**Topic:** Project [Raijin](https://raijin.ucar.edu/) & [UXarray](https://uxarray.readthedocs.io/)
+
+*What did your group do?*
+
+The UXarray devs had a chance to discuss use cases and get feedback from the community members on UXarray. We branched into subgroups for different work ideas and coded with our users. We noticed a significant bug and fixed that throughout the end of the day.
+
+*Was it a fruitful experience?*
+
+Very much!
+
+*What worked for your group and what didn't work?*
+
+Sitting together with the users and listening to their expectations was a great opportunity! Since we had multiple UXarray folks as facilitators, things worked quite well for us. That wouldn't happen if we didn't have that many helpers (as a side note for the future)
+
+*Would it be useful to redo the  topic again?  What would you do differently the second time around?*
+
+Yes, after some time. We'd add one or two work idea of our own group as well as those came from the community. There were bugs related to
+1. UXarray's MPAS data ingest (attributes not being parsed) with the dual mesh only (see PRs [#274](https://github.com/UXARRAY/uxarray/pull/274) and [#275](https://github.com/UXARRAY/uxarray/pull/275)), and
+2. face area calculations (see PR [#283](https://github.com/UXARRAY/uxarray/pull/283))
+
+
+#### Falko Judt (MMM):
+
+**Topic** Extract Tropical Cyclone Vitals from MPAS
+
+*What did your group do?*
+
+ Free discussion. Mostly scientist asking questions to the GeoCAT/UXarray experts. We didn't write any code because nobody had a project ready.
+
+*Was it a fruitful experience?*
+
+ Yes.
+
+*What worked for your group and what didn't work?*
+
+Discussion was great. Also just meeting some people face to face was a plus.
+
+*Would it be useful to redo the  topic again?  What would you do differently the second time around?*
+
+Yes, maybe in 1/2 year or so? Could tell people to have some project ready to get into some hands-on coding.
+
+
+#### Katie Dagon (CGD):
+
+**Topic**: Visualizing & Analyzing CAM-SE output in Python
+
+*What did your group do? Write code? discussions?*
+
+We started with open discussion and sharing our experiences with the overall topic (CAM-SE analysis/visualization).
```

Comment:
Perhaps add a definition of CAM-SE: Community Atmosphere Model (CAM) with the spectral element (SE) grid.